scss/_buttons.scss

```diff
@@ -68,6 +68,7 @@
   }
 
   // Boosted mod: custom styles for .btn-check
+
```

Comment:
Intentional ?

Reply:
Yes when I was reading the code I thought it was the beginning of a ` // Boosted mod // End mod` block. Maybe it's more "No custom styles for .btn-check`. Feels like a duplicate of line 53.

Reply:
Ah yes, you're right, feels like dupe !

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1507/commits/97c768c9e2aff3469ca0667288d2dab045a1d5ef after a comparison with Bootstrap's _main_ branch.